js/stdlib/ts/ETH_like.ts

```diff
@@ -276,24 +276,39 @@ const getMinBlock = (logs: any[]) =>
       ? (x.logIndex < acc.logIndex ? x : acc)
       : (x.blockNumber.toString() < acc.blockNumber.toString() ? x : acc), logs[0]);
 
+const getMaxBlock = (logs: any[]) =>
+  logs.reduce((acc: Log, x: Log) =>
```

Comment:
Another `argmax`

Reply:
I don't think these ETH functions translate well to `argmax`, I translated the ALGO ones to use `argmax`

Reply:
Sounds good